setup.py

```diff
@@ -4,11 +4,14 @@
 from setuptools import setup, find_packages
 from os         import path
 
+import cvmfs
+
+
 readme_path = path.join(path.dirname(__file__), 'README')
 
 setup(
-  name="python-cvmfsutils",
-  version="0.2.0",
+  name=cvmfs.package_name,
+  version=cvmfs.version,
```

Comment:
Nope, it doesn't. If you have a version installed already it works fine. But if you install `cvmfsutils` from scratch it fails. See this [StackOverflow post](http://stackoverflow.com/questions/2058802/how-can-i-get-the-version-defined-in-setup-py-setuptools-in-my-package) in the last paragraph of the top-answer.